app/src/main/java/org/mifos/mobile/ui/savings_account_withdraw/SavingsAccountWithdrawScreen.kt

```diff
@@ -0,0 +1,241 @@
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.component.MifosOutlinedTextField
+import org.mifos.mobile.core.ui.component.MifosProgressIndicator
+import org.mifos.mobile.core.ui.component.MifosTopBar
+import org.mifos.mobile.core.ui.component.NoInternet
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+import org.mifos.mobile.models.accounts.savings.SavingsWithAssociations
+import org.mifos.mobile.utils.Constants
+import org.mifos.mobile.utils.Network
+import org.mifos.mobile.utils.getTodayFormatted
+
+
+@Composable
+fun SavingsAccountWithdrawScreen(
+    viewModel: SavingsAccountWithdrawViewModel = hiltViewModel(),
+    navigateBack: (withdrawSuccess: Boolean) -> Unit,
+    withdraw: (String) -> Unit,
+) {
+    val uiState by viewModel.savingsAccountWithdrawUiState.collectAsStateWithLifecycle()
+    val savingsWithAssociations by viewModel.savingsWithAssociations.collectAsStateWithLifecycle()
+
+    SavingsAccountWithdrawScreen(
+        uiState = uiState,
+        savingsWithAssociations = savingsWithAssociations,
+        navigateBack = navigateBack,
+        withdraw = withdraw,
+    )
+}
+
+@Composable
+fun SavingsAccountWithdrawScreen(
+    uiState: SavingsAccountWithdrawUiState,
+    savingsWithAssociations: SavingsWithAssociations?,
+    navigateBack: (withdrawSuccess: Boolean) -> Unit,
+    withdraw: (String) -> Unit,
+) {
+    val context = LocalContext.current
+
+    Column(modifier = Modifier.fillMaxSize()) {
+        MifosTopBar(
+            navigateBack = { navigateBack(false) },
+            title = { Text(text = stringResource(id = R.string.withdraw_savings_account)) }
+        )
+
+        Box(modifier= Modifier.weight(1f)) {
+            SavingsAccountWithdrawContent(
+                savingsWithAssociations = savingsWithAssociations,
+                withdraw = withdraw
+            )
+
+            when (uiState) {
+                is SavingsAccountWithdrawUiState.Initial -> {
```

Comment:
Remove this state, we don't need it.

Reply:
Then what can we use to show the UI only, and not any Success, Error or Loading state. We have just a screen to show the data, and Success, Error and Loading will be called after withdraw button is clicked. Just to show data UI, I have used Initial state.